service/service.go

```diff
@@ -71,22 +71,30 @@ func NewService(options Options, registry *plugin.Registry) *Service {
 
 func (s *Service) Start() error {
 	if s.options.Log == "console" {
+		log.SetOutput(os.Stdout)
 		log.SetFormatter(&log.TextFormatter{})
 	} else if s.options.Log == "syslog" {
-		hook, err := logrus_syslog.NewSyslogHook("", "", syslog.LOG_INFO, "")
+		devNull, err := os.OpenFile("/dev/null", os.O_WRONLY, 0)
+		if err != nil {
+			panic(err)
```

Comment:
It seems to me that in that case the right solution would be to update the logging parameter you start your vulcand with to remove syslog and use something else explicitly. Falling back to console is not obvious, for the user specified syslog for a reason and they will be scratching their head wondering why it is not working.